cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/TransformExecutorFactory.java

```diff
@@ -142,8 +142,9 @@ public TransformExecutorFactory(JobContext hadoopContext, PipelinePluginInstanti
       addTransformation(pipeline, stageName, transformations, transformErrorSinkMap);
     } catch (Exception e) {
       // Catch the Exception to generate a User Error Log for the Pipeline
-      PIPELINELOG.error("Failed to start Stage '{}'. Please review your Pipeline configuration" +
-                          " and check System logs for more details.", stageName, Throwables.getRootCause(e));
+      PIPELINELOG.error("Failed to start Stage '{}' with error '{}' Please review your Pipeline configuration" +
```

Comment:
Hmmm now I think that enclosing the entire error message in '...' is not good, because it may contain quotes in itself. Better say, "... with error: <message>. Please ..."